pkg/node_reputation/node_reputation_handler.go

```diff
@@ -0,0 +1,280 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package nodereputation
+
+import (
+	"database/sql"
+	"strconv"
+
+	"golang.org/x/net/context"
+	//"storj.io/storj/protos/node_reputation"
+	proto "storj.io/storj/protos/node_reputation"
+)
+
+// Server is a struct
+type Server struct{}
+
+// UpdateReputation in handler
+func (s *Server) UpdateReputation(ctx context.Context, in *proto.NodeUpdate) (*proto.UpdateReply, error) {
+	db, err := SetServerDB("./Server.db")
```

Comment:
maybe have db be a field of `s` (`*Server`) so you don't have to open it and close it each operation?